design/proposals/spike-cli-dockerless.md

```diff
@@ -0,0 +1,209 @@
+| Python CLI | More or less at level of Go CLI in terms of functionality coverage. |
+
+The following are mappings of functionalities for both Go and Python CLIs.
+
+![golang-cli-mapping](../images/golang-cli-mapping.png)
+
+![python-cli-mapping](../images/python-cli-mapping.png)
+
+|            | Quality                                                 |
+| ---------- | ------------------------------------------------------- |
+| Ruby CLI   | Widely used<br />***Status***: Stable                   |
+| Go CLI     | Not "community" level certified <br />***Status***: POC |
+| Python CLI | ***Status***: Community/alpha level                     |
+
+|            | UX                                                           |
+| ---------- | ------------------------------------------------------------ |
+| Ruby CLI   | Unintuitive and inconsistent CLI command construction. See list below for thorough explanations. |
+| Go CLI     | Similar to Ruby CLI                                          |
+| Python CLI | More straightforward approach. See list below. For example, when setting a Conjur secret,  `conjur-cli variable set my_id supersecretvarid` |
+
+#### Ruby/Go CLI UX
+
+The following are examples of flaws in our current Ruby/Go CLI UX
+
+*Verb/noun placement*
+
+Our Ruby and Go CLIs currently are inconsistent in how the command is constructed. It is not consistent if the noun or the verb comes first. For example, we currently have `conjur variable` and `conjur show` 
+
+*Unintuitive command construction*
+
+When running `conjur variable value`, a secret value will be returned to you but when looking at just the command, it is unclear what the purpose of the command is. For a more straightforward approach, the command for display a secret value could be `conjur get <variable_id>` or `conjur show <variable_id>`.  
+
+Another example of uninuitive command construction would be trying to understand what the difference between `conjur variable value`  is and `conjur variable values` without using the `--help` flag.
+
+*Permitted edge cases*
+
+We are currently not blocking the chaining of illogical flagging. For example, `conjur policy load --delete --replace foo foo.yml` does not lead to an error.
+
+#### Python CLI UX
+
+The Python CLI has a more straightforward experience. For example, when setting and getting variables, the user will run
+
+```
+conjur-cli variable set my_var_id mysupersecretvar
+conjur-cli variable get my_var_id
+```
+
+For policy maintaince the outcome of the command is very clear. Unlike the Ruby/Go CLI, there is no chance of an unpredictable outcome because running `replace` and `delete`  would lead to an error.
+
+```bash
+conjur-cli policy apply root ./path_to_policy.yml
+conjur-cli policy replace root ./path_to_policy.yml
+conjur-cli policy delete root ./path_to_policy.yml
```

Comment:
Add a comment about the `policy delete` does **not** mean deleting the policy.
This is a severe UX flaw.